tests/cupy_tests/testing_tests/test_helper.py

```diff
@@ -78,3 +80,140 @@ def test_forbidden_error(self):
             helper._check_cupy_numpy_error(
                 self, cupy_error, cupy_tb,
                 numpy_error, numpy_tb, accept_error=False)
+
+
+def make_func(np_result, cp_result):
```

Comment:
It is too generalized to understand. How about making simplified functions like `numpy_error` and `cupy_error`?